Makefile

```diff
@@ -67,66 +67,66 @@ install: composer.phar
 define cakephp3
 build-cakephp-$(VERSION): install
 	cd $(CAKEPHP_SOURCE_DIR) && git checkout -f $(TAG)
-	cd $(CAKEPHP_SOURCE_DIR) && $(PHP7) $(COMPOSER) update
+	cd $(CAKEPHP_SOURCE_DIR) && $(PHP7) $(COMPOSER) update -n
 	mkdir -p $(BUILD_DIR)/cakephp/$(VERSION)
 	cp -r static/assets/* $(BUILD_DIR)/cakephp/$(VERSION)
 
-	$(PHP8) bin/apitool.php generate --config cakephp3 --version $(VERSION) \
+	$(PHP8) bin/apitool.php generate --config cakephp3 --version $(VERSION) --tag $(TAG) \
```

Comment:
What's the difference between tag and version?

Reply:
The tag is needed to create a valid link to github.
We can't just link to https://github.com/cakephp/cakephp/blob/4.3/src/Datasource/Locator/AbstractLocator.php#L32-L32 because that tag is not present.
But with the given `tag` from the build command we exactly know which tagged version was used to generate the docs and therefore can Link directly to it.
<img width="289" alt="image" src="https://user-images.githubusercontent.com/9105243/161386965-0fc22d76-0843-4d12-974f-0af91afbe87e.png">